example/traefik/config.minimal.yml

```diff
@@ -0,0 +1,61 @@
+###############################################################
```

Comment:
This config will end up in a suite with tests instead.
I'll do it for you in the next commit if you want. This will make Traefik officially supported in Authelia project!

Reply:
Thanks! Do you want me to remove this then? Or should I leave it so you can move it wherever it needs to go?

Reply:
I will do it. Thanks.